.bluemix/pipeline-DEPLOY.sh

```diff
@@ -10,46 +12,51 @@ fi
 echo "Using manifest file: $MANIFEST"
 echo "Using prefix: $PREFIX"
 
-# Create CF services
-cf create-service conversation free insurance-bot-conversation
-cf create-service cloudantNoSQLDB Lite ${PREFIX}insurance-bot-db
-# Set app's env vars
-domain=".mybluemix.net"
-case "${REGION_ID}" in
-  ibm:yp:eu-gb)
-    domain=".eu-gb.mybluemix.net"
-  ;;
-  ibm:yp:au-syd)
-  domain=".au-syd.mybluemix.net"
-  ;;
-  ibm:yp:eu-de)
-  domain=".eu-de.mybluemix.net"
-  ;;
-esac
-if ! cf app $CF_APP; then
-  cf push $CF_APP -n $CF_APP -f $MANIFEST --no-start
+# Create services
+bx service create conversation free insurance-bot-conversation
+bx service create cloudantNoSQLDB Lite ${PREFIX}insurance-bot-db
+bx service create appid "Graduated tier" insurance-bot-appid
```

Comment:
@data-henrik 

we had ${PREFIX} for the db to make sure we did not mix the dev db with the master db so both deployments are isolated.

Any impact having dev+master using the same App ID?

I'm surprised we use the same conversation service - if I change the conversion flow in dev, master will not know how to work with the updated workspace given it is shared

Reply:
Ideally, we have different instances for all three services. But that is extra to the integration. How about we address that with a separate defect?